medusa/post_processor.py

```diff
@@ -1008,12 +996,12 @@ def process(self):
                     self._log(u'New file is a PROPER, marking it safe to replace')
                     self.flag_kodi_clean_library()
                 else:
-                    _, preferred_qualities = show.current_qualities
-                    if new_ep_quality not in preferred_qualities:
+                    allowed, preferred = show.current_qualities
+                    if not common.Quality.is_higher_quality(old_ep_quality, new_ep_quality, allowed, preferred):
```

Comment:
@medariox don't change something that you don't understand. You can't change this. If you don't know why is because you don't understand the quality system